CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+1.5.0 --- 2021-07-19
+--------------------
+
+* Added support for django 3.0, 3.1 and 3.2
```

Comment:
We don't need Django 3.0 or 3.1, right? Can't we simplify by just keeping Django 3.2?